src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.ql

```diff
@@ -14,6 +14,9 @@
  * @query-version 1.0
  * @precision high
  * @platform Desktop
+ * @tags correctness
+ *       wddst

```

Comment:
What is this field? Looks like extra tab.

Reply:
See https://github.com/microsoft-sdl/codeql-queries#required-metadata-fields-for-security-queries.  The extra spacing is coding style for the SDL repo.